test/lit/passes/gufa-closed-open.wast

```diff
@@ -12,15 +12,15 @@
 
  ;; OPEND:      (type $2 (func (param i32)))
 
- ;; OPEND:      (import "fuzzing-support" "call-ref-catch" (func $external-caller (type $0) (param funcref)))
+ ;; OPEND:      (import "outside" "call-ref-catch" (func $external-caller (type $0) (param funcref)))
  ;; CLOSE:      (type $0 (func (param funcref)))
 
  ;; CLOSE:      (type $1 (func))
 
  ;; CLOSE:      (type $2 (func (param i32)))
 
- ;; CLOSE:      (import "fuzzing-support" "call-ref-catch" (func $external-caller (type $0) (param funcref)))
- (import "fuzzing-support" "call-ref-catch" (func $external-caller (param funcref)))
+ ;; CLOSE:      (import "outside" "call-ref-catch" (func $external-caller (type $0) (param funcref)))
+ (import "outside" "call-ref-catch" (func $external-caller (param funcref)))
```

Comment:
Why this change?

Reply:
Ah, this API now has a meaning :smile: 

And that meaning is "return i32, receive funcref", but we were not declared as such. The fuzzer errors on using this as initial content because of the mismatch.

(Another fix could have been to fix the return type and add a drop on the call.)